### ISSUE TITLE
Blank blocks data in JSON makes the editor not visible and clicking or dragging does nothing

### ISSUE THREAD

**Issue by odoookentang** [codex-team/editor.js#2493]
## Describe a bug.
The editor is not visible and nothing works. clicking or dragging anywhere the editor was supposed to be, it returns this error:
```
editorjs@latest:83  Uncaught TypeError: Cannot read properties of undefined (reading 'holder')
    at di.redactorClicked (editorjs@latest:83:6323)
    at HTMLDivElement.<anonymous> (editorjs@latest:83:2231)
redactorClicked @ editorjs@latest:83
(anonymous) @ editorjs@latest:83
```

## Steps to reproduce:
1. use `editor.render` function on this example data `{"time":1695567751407,"blocks":[],"version":"2.28.0"}`

## Expected behavior:
The editor continues to work normally, for example displaying a placeholder.

## Screenshots:
![image](https://github.com/codex-team/editor.js/assets/72538426/1d29daa7-15af-4e83-86a0-eb223758c759)

## Device, Browser, OS:
Microsoft Edge 117.0.2045.36 (Official build) (64-bit), Windows

## Editor.js version:
2.28.0

## Plugins you use with their versions:
(plugins do not affect this issue.)

**Comment by Darcrandex** [comment]
+1

**Comment by neSpecc** [comment]
It's fixed in 2.29 which is available via `2.29.0-rc.1`

https://github.com/codex-team/editor.js/releases
